common_utils/tests/test_commands.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management import call_command
+from django_ilmoitin.models import NotificationTemplate
+
+
+def test_command_generate_notifications_from_templates_generates_notifications():
+    assert NotificationTemplate.objects.count() == 0
+
+    call_command("generate_notifications_from_templates")
+
+    assert NotificationTemplate.objects.count() > 0
```

Comment:
Any other cases I should test?

Reply:
If you want I guess you could make it a tiny bit more explicit by testing that correct amount of templates and translations are generated (similarly how it was in the previous tests)

```
    #from django.apps import apps
    NotificationTemplateTranslation = apps.get_model(  # noqa: N806
        "django_ilmoitin", "NotificationTemplateTranslation"
    )

    assert NotificationTemplate.objects.count() == 2
    assert NotificationTemplateTranslation.objects.count() == 6
```